Source/WebKit/UIProcess/Extensions/WebExtension.h

```diff
@@ -214,8 +214,10 @@ class WebExtension : public API::ObjectImpl<API::Object::Type::WebExtension>, pu
 
     NSArray *errors();
 
+#ifdef __OBJC__
     _WKWebExtension *wrapper() const { return (_WKWebExtension *)API::ObjectImpl<API::Object::Type::WebExtension>::wrapper(); }
```

Comment:
Would be nice to use `dynamic_objc_cast<_WKWebExtension>(...)` here from `<wtf/cocoa/TypeCastsCocoa.h>`.

(Technically it's defined in `<wtf/RetainPtr.h>`, but it may move in the future.)

Reply:
Okay, `(id)` cast here is fine.

Reply:
That caused an error since the ObjC class is a forward declaration, and the `isKindOfClass:` check needs the ObjC class header included to do the `[T class]` call.

Reply:
I'll keep these as simple casts.

Reply:
Okay.